src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -615,11 +614,8 @@ public static AssetSpecification getAssetByNameFromDataset(Dataset dataset, Stri
                     "This dataset does not have an asset specification with name: " + assetName));
   }
 
-  public List<UUID> getSourceDatasetIdsFromSnapshotRequest(
-      SnapshotRequestModel snapshotRequestModel) {
-    return getSourceDatasetsFromSnapshotRequest(snapshotRequestModel).stream()
-        .map(Dataset::getId)
-        .collect(Collectors.toList());
+  public List<UUID> getDatasetsIds(List<Dataset> datasets) {
+    return datasets.stream().map(Dataset::getId).toList();
```

Comment:
I feel like generally we aren't supporting the multiple datasets for a snapshot anymore, is that right?